WatchApp Extension/Controllers/HUDInterfaceController.swift

```diff
@@ -67,14 +67,16 @@ class HUDInterfaceController: WKInterfaceController {
             }
         }())
 
-        glucoseLabel.setText("---")
-        glucoseLabel.setHidden(false)
-        if let glucose = activeContext.glucose, let glucoseDate = activeContext.glucoseDate, let unit = activeContext.preferredGlucoseUnit, glucoseDate.timeIntervalSinceNow > -LoopCoreConstants.inputDataRecencyInterval {
-            let formatter = NumberFormatter.glucoseFormatter(for: unit)
-
-            if let glucoseValue = formatter.string(from: glucose.doubleValue(for: unit)) {
-                let trend = activeContext.glucoseTrend?.symbol ?? ""
-                glucoseLabel.setText(glucoseValue + trend)
+        if date != nil {
```

Comment:
Strange...I had thought my latest change included this `if date != nil`... I'm puzzled.  But thanks.

Reply:
Hmm, I'm not seeing this as a change I made. I think this was brought in by me merging in the latest dev. Maybe you're looking at changes that include the merge itself.

Reply:
FYI - I do not see this line as a difference in my GitHub PR review.

Reply:
I no longer see it as a diff either. It was there 42 minutes ago.  Sigh. This is the second "UI strangeness" in github in this PR that I've seen now. 🤷 
Either way, the check should be in there.